### PR TITLE
CI: use GitHub Actions for Pull Requests / Push tests

### PR DESCRIPTION
**Pull request by vltansky**
I guess we should use one CI, for easier maintenance.

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```
Comment (roblarsen) [inline .github/workflows/test.yaml:1]: Can you remove the comment here?

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - uses: actions/checkout@v2
+      - name: Run tests ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - run: npm test
```
Comment (roblarsen) [inline .github/workflows/test.yaml:37]: newline

Comment (vltansky) [reply .github/workflows/test.yaml:37]: Didn't understand this ?

Comment (roblarsen) [reply .github/workflows/test.yaml:37]: there's no newline at the end of the file.

**Comment by vltansky** [comment]
Done

**Comment by vltansky** [comment]
You should share your list of things :) Maybe other people would want to work on it (Including me)

**Comment by roblarsen** [comment]
@vltansky  I will put together some issues. I just had the thought recently during the 8.0.0 release. I also thought about moving away from grunt so that's another area where I'll generate some issues.

**Pull request merged**